tex/preamble.tex

```diff
@@ -190,7 +190,7 @@
 %------------------------------------------------------+
 % Hyperlinks, bookmarks, theorems and cross-references |
 %------------------------------------------------------+
-\usepackage[hyperfootnotes=false]{hyperref}
+\usepackage{hyperref}
```

Comment:
Remove this line

Reply:
`mybibformat.sty` also does his own `'\usepackage{hyperref}`

```tex
  429  %---------------+
  430: % Load hyperref |
  431  %---------------+
  432  % We need to do this here after patching
  433  \makeatletter
  434: %  FIXME: This doesn't always work and we need to manually load hyperref when
  435  % there are errors stating that it was not loaded
  436: \AtEndPreamble{\@ifpackageloaded{hyperref}{}{\usepackage{hyperref}}}
  437  \makeatother
  438  ```

Reply:
Removed that line and `Undefined control sequence. \hypersetup`

Reply:
Fair enough. The following works (at least locally):
```
\PassOptionsToPackage{hyperfootnotes=false, colorlinks=true, allcolors=blue,
linktoc=page,
pdfstartview={xyz null null 1}, pdfcreator={vim latex},
pdfsubject={machine learning},
pdftitle={tesis para la carrera de licenciatura en matematica aplicada},
pdfauthor={juan mateo de monasterio},
pdfcreator={juan mateo de monasterio},
pdfkeywords={machine learning}
}{hyperref}
```